dts/riscv/starfive/jh7110-visionfive-v2.dtsi

```diff
@@ -133,9 +133,14 @@
 		};
 	};
 
-	ram0: memory@8000000 {
+	l2lim: memory@8000000 {
 		device_type = "memory";
-		reg = <0x0 0x8000000 0x1 0x0>;
+		reg = <0x0 0x8000000 0x0 0x200000>;
+	};
+
+	dtim: memory@01100000 {
+		device_type = "memory";
+		reg = <0x0 0x01100000 0x0 0x2000>;
```

Comment:
Did you mean `S7 Hart0 DTIM` here? As per the JH7110 TRM here: [https://doc-en.rvspace.org/JH7110/TRM/JH7110_TRM/u74_memory_map.html](https://doc-en.rvspace.org/JH7110/TRM/JH7110_TRM/u74_memory_map.html) the address space starts at `0x00_0110_1000`.

Reply:
Yes and that is an erratum in the TRM.

From reverse engineering the mask ROM and reading out the DTIM, I know that it really starts at the address I wrote here. Then the size in the TRM also fits with the end address.

Reply:
Agreed on the TRM errata, needs correction on starfive's part. This is from sifive's official books: [https://starfivetech.com/uploads/u74mc_core_complex_manual_21G1.pdf](https://starfivetech.com/uploads/u74mc_core_complex_manual_21G1.pdf), Page number `96` says:  

> 0x0100_0000 0x0100_1FFF RWX A S7 Hart 0 DTIM (8 KiB)

Still not the address it mentions here and also you intend to mention.

Reply:
I would think that's the reference design.

All I can say is that I dumped the memory area I put in here and it had the expected data. You can do that as well to verify, e.g. by adding some code to oreboot.

Reply:
We are still trying to understand the significance of this `dtim` node in the zephyr context here.

Provide some steps to verify the **S7 Hart 0 DTIM** address.

Reply:
There are two ways coming to my mind:
1. dump and reverse engineer the mask ROM, e.g. via U-Boot + TFTP
2. read out from the address I provide at runtime, e.g. via oreboot or U-Boot